runtime/autoload/remote/host.vim

```diff
@@ -162,9 +164,10 @@ function! s:RegistrationCommands(host) abort
       call add(lines, "      \\ ".string(spec).",")
     endfor
     call add(lines, "     \\ ])")
+    call add(registered, path)
   endfor
   echomsg printf("remote/host: %s host registered plugins %s",
-        \ a:host, string(map(copy(paths), "fnamemodify(v:val, ':t')")))
+        \ a:host, string(map(registered, "fnamemodify(v:val, ':t')")))
```

Comment:
`map()` modifies list in-place. It is a good practice to not use `copy(list)` or `list[:]` *only* when you need in-place modification in `:call` or when you need super optimization, not when it is harmless in the current state of code because it is only harmless in the *current* state, not outside of `:call` because it is hard to mention side-effects otherwise and not when your goal is something other then making code that is called really often run a bit faster (I needed such optimizations only for a few thousand calls).